### DISCUSSION TITLE
Regarding iperf

### DISCUSSION THREAD

**Discussion by Bayrun** [esnet/iperf#1235] [Q&A]
Hello, please help me with that matter:
I downloaded iperf3 and searched for Public iPerf3 servers, I found this server in your list "iperf.par2.as49434.net"
and accidentally I wrote in cmd that: "iperf.exe -s iperf.par2.as49434.net" , and allowed the windows defender firewall rule for private and public communication that popped up.
and it said something about listening at port 5XXX. (can't remember exact numbers)
then I realized "-s" means I'm making my computer a server, and I wrote the public server address there!!!
am I a server now?? I did something wrong?
As you can tell I know absolutely NOTHING about these stuff :p
but I'm worried and I need your help to tell me how to undo this.
Thanks a lot :)

**Comment by MrM40** [comment]
Just close down iperf and the "server" is down again (if you ever were acting as a server). it doesn't run in the background as a service or something like it that I assume you might fear.

**Reply by Bayrun** [reply]
Thank you Mr :) 👍
